packages/dashboard/src/dashboard-actions/createWidget.ts

```diff
@@ -0,0 +1,13 @@
+import { DashboardConfiguration, Widget } from '../types';
+import { concatWidgets } from '../util/dashboardConfiguration';
+
+export const createWidget = ({
+  dashboardConfiguration,
+  widgets,
+}: {
+  dashboardConfiguration: DashboardConfiguration;
+  widgets: Widget[];
+}): DashboardConfiguration => {
+  dashboardConfiguration = concatWidgets(dashboardConfiguration, widgets);
```

Comment:
no need to reassign the variable here since concatWidgets returns a dashboard configuration.